### PR TITLE
[Synthetics] Add support for global variables from test

### PR DESCRIPTION
**Pull request by romainberger**

### CODE REVIEW

**Review by sdeprez — DISMISSED** [review]

**Comment by romainberger** [comment]
Please wait before merging, it needs a fix from https://github.com/DataDog/datadog-api-spec/pull/759

**Comment by romainberger** [comment]
Good to merge when approved.

File: datadog/resource_datadog_synthetics_global_variable_test.go
Comment (therve) [inline datadog/resource_datadog_synthetics_global_variable_test.go:299]: Should we use new assertion format in tests?

File: datadog/resource_datadog_synthetics_global_variable_test.go
Comment (therve) [inline datadog/resource_datadog_synthetics_global_variable_test.go:312]: If you can fix indentation that'd be great.

File: datadog/resource_datadog_synthetics_global_variable.go
Comment (therve) [inline datadog/resource_datadog_synthetics_global_variable.go:56]: Please add description to all fields defined below.

**Review by therve — CHANGES_REQUESTED** [review]
Some minor comments, otherwise looks good.

**Review by therve — APPROVED** [review]

**Pull request merged**
